src/epgsql_wire.erl

```diff
@@ -209,6 +209,6 @@ encode_list(L) ->
     <<(byte_size(Bin)):?int32, Bin/binary>>.
 
 encode_standby_status_update(ReceivedLSN, FlushedLSN, AppliedLSN) ->
-    {MegaSecs, Secs, MicroSecs} = erlang:now(),
```

Comment:
erlang:now() is deprecated http://erlang.org/doc/apps/erts/time_correction.html and it throws compilation warnings in 18 and 19 Erlang's
http://tryerl.seriyps.ru/#id=4df3